From: Alexander Larsson <alexl@redhat.com>
Date: Fri, 9 Sep 2016 09:44:54 +0200
Subject: builder: Bump manifest version to trigger rebuild with new mtimes

To make sure we're not using old caches with the previous mtime
timestamps encoded we bump the manifest version to force a rebuild.

Origin: upstream, 0.6.10, commit:5ef43f6ee3909d077a3cbf3b90abcd9e01e1e930
---
 builder/builder-manifest.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/builder/builder-manifest.h b/builder/builder-manifest.h
index 6917c48..4dc3a95 100644
--- a/builder/builder-manifest.h
+++ b/builder/builder-manifest.h
@@ -37,7 +37,7 @@ typedef struct BuilderManifest BuilderManifest;
 #define BUILDER_IS_MANIFEST(obj) (G_TYPE_CHECK_INSTANCE_TYPE ((obj), BUILDER_TYPE_MANIFEST))
 
 /* Bump this if format changes in incompatible ways to force rebuild */
-#define BUILDER_MANIFEST_CHECKSUM_VERSION "1"
+#define BUILDER_MANIFEST_CHECKSUM_VERSION "2"
 #define BUILDER_MANIFEST_CHECKSUM_CLEANUP_VERSION "1"
 #define BUILDER_MANIFEST_CHECKSUM_FINISH_VERSION "2"
 #define BUILDER_MANIFEST_CHECKSUM_PLATFORM_VERSION "1"
